src/server/cluster/cluster_family.cc

```diff
@@ -660,26 +660,6 @@ static string_view StateToStr(MigrationState state) {
   return "UNDEFINED_STATE"sv;
 }
 
-static uint64_t GetKeyCount(const SlotRanges& slots) {
```

Comment:
where is the bug? where did we count all keys and not only the migrated keys?

Reply:
when migration is finished and we start to process new requests the number of keys will be changed and we will show incorrect statistic

Reply:
Oh I see